tests/pipelines/controlnet/test_controlnet.py

```diff
@@ -475,12 +475,12 @@ def test_inference_multiple_prompt_input(self):
         sd_pipe.set_progress_bar_config(disable=None)
 
         inputs = self.get_dummy_inputs(device)
-        inputs["prompt"] = [inputs["prompt"], inputs["prompt"]]
-        inputs["image"] = [inputs["image"], inputs["image"]]
+        inputs["prompt"] = [inputs["prompt"], inputs["prompt"], inputs["prompt"], inputs["prompt"]]
```

Comment:
instead of update this test, can you add a new one?
I think we also accept inputs like this

Reply:
it can be in the same test, just add a few lines to test new input format

Reply:
OK I will add tonight later